virtualizarr/zarr.py

```diff
@@ -368,9 +368,14 @@ def metadata_from_zarr_json(filepath: Path) -> tuple[ZArray, list[str], dict]:
         for codec in metadata["codecs"]
         if codec["name"] not in ("transpose", "bytes")
     ]
-    compressor, *filters = [
+    num_codec_configs = [
         _configurable_to_num_codec_config(_filter) for _filter in all_codecs
     ]
+    # Uncompressed variables may have no codecs
+    if num_codec_configs:
```

Comment:
Is `len(num_codec_configs) > 0` better? Or `not empty(num_codec_configs)`?

Reply:
Does this actually cover all cases? What if you have an uncompressed variable that does have other codecs?